documents/documents.html

```diff
@@ -360,7 +361,49 @@ <h2>Label Inputs</h2>
             >
             </iframe>
           </div>
-         
+          <p class="fs-2 mt-4 font-weight-bold">Checkbox inputs</p>
+          <p class="pt-1 ml-1">
+            Checkbox is used to select one or more option from number of
+            choices.Use <span class="orange-theme-color">form-check-input</span> for basic styling
+          </p>
+          <div class="input-container mt-2 ml-1">
+            <input
+              id="checkbox-checked"
+              class="form-check-input"
+              type="checkbox"
+              checked
+            />
+            <label for="checkbox-checked">Checked</label>
+
+            <input
+              id="checkbox-unchecked"
+              class="form-check-input"
+              type="checkbox"
+            />
+            <label for="checkbox-unchecked"> Unchecked</label>
+
+            <input
+              id="checkbox-disabled"
+              class="form-check-input"
+              type="checkbox"
+              disabled
+            />
+            <label for="checkbox-disabled"> disabled</label>
+          </div>
+          <div class="mt-2 ml-1 code-snippet">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%2821%2C23%2C24%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=100px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cinput%2520id%253D%2522checkbox-checked%2522%2520class%253D%2522form-check-input%2522%2520type%253D%2522checkbox%2522%2520checked%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522checkbox-checked%2522%253EChecked%253C%252Flabel%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cinput%2520id%253D%2522checkbox-unchecked%2522%2520class%253D%2522form-check-input%2522%2520type%253D%2522checkbox%2522%252F%253E%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522checkbox-unchecked%2522%253EUnchecked%253C%252Flabel%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cinput%2520id%253D%2522checkbox-disabled%2522%2520class%253D%2522form-check-input%2522%2520type%253D%2522checkbox%2522%2520disabled%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522checkbox-disabled%2522%253Edisabled%253C%252Flabel%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
+              style="
+                width: 797px;
+                height: 241px;
```

Comment:
you may use `rem` instead of `px` for more than `10px` value.

Reply:
Its coming from carbon, I have used rem in my code